src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawing {
+    private String colorfigure;
+    private double calcutateArea;
+
+    public double getcalcutateArea() {
+        return calcutateArea;
+    }
```

Comment:
better create a separate interface with the method getArea() and calculate the area for each type of figure